eng/tools/generate_api_report/src/main.rs

```diff
@@ -121,7 +121,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     }
 
     // Create the package_name.rust.json in the review/ folder
-    let output_path_str = review_folder_path.join(format!("{}.rust.json", package_name));
+    let output_path_str = review_folder_path.join(format!("{}_rust.json", package_name));
```

Comment:
why this change?

Reply:
@chidozieononiwu @praveenkuttappan 
We decided on `x.rust.json` for the JSON file(for Rust) to be uploaded in the portal during the development in the staging arena, are there any implications because of this change?

Reply:
I think you should keep it as ` .rust.json`